### DISCUSSION TITLE
Recreation of 1-loop diagram in FeynCalc

### DISCUSSION THREAD

**Discussion by acassem** [FeynCalc/feyncalc#239] [Q&A]
Hello, first I am completely new to the FeynCalc and Mathematica scene. Now my question/help is that I am trying to re-create the amplitude in equation (58) in https://arxiv.org/abs/0708.1701 and nothing seems to be working. These are graviton scatterings where I inputted the Feynman rules myself (given in the appendix) and then I entered in their 1PI equation (57). Again, no matter what I do I do not recover the same result as they do.

Maybe the way I am writing the rules are wrong, or my IR-limit is wrong (doubtful but I am not ruling it out) or something else entirely. Here is the code (the best I could format it)
Here are the Feynman rules:
`TAUk1k2m1 = ((-I*\[Kappa])/
   2)*(FVD[Subscript[k, 1], \[Mu]] FVD[Subscript[k, 2], \[Nu]] + 
    FVD[Subscript[k, 1], \[Nu]] FVD[Subscript[k, 2], \[Mu]] - 
    MTD[\[Mu], \[Nu]]*((SPD[Subscript[k, 1], Subscript[k, 2]]) - 
       Subscript[m, 1]^2))
TAUk3k4m2 = ((-I*\[Kappa])/
   2)*(FVD[Subscript[k, 3], \[Rho]] FVD[Subscript[k, 4], \[Sigma]] + 
    FVD[Subscript[k, 3], \[Sigma]] FVD[Subscript[k, 4], \[Rho]] - 
    MTD[\[Rho], \[Sigma]]*((SPD[Subscript[k, 3], Subscript[k, 4]]) - 
       Subscript[m, 2]^2))
gravprop1 = (I/(
   2*SPD[q]))*(MTD[\[Mu], \[Alpha]] MTD[\[Nu], \[Beta]] + 
    MTD[\[Mu], \[Beta]] MTD[\[Nu], \[Alpha]] - 
    MTD[\[Mu], \[Nu]] MTD[\[Alpha], \[Beta]])
gravprop2 = (I/(
   2*SPD[q]))*(MTD[\[Gamma], \[Rho]] MTD[\[Delta], \[Sigma]] + 
    MTD[\[Gamma], \[Sigma]] MTD[\[Delta], \[Rho]] - 
    MTD[\[Gamma], \[Delta]] MTD[\[Rho], \[Sigma]])`

Here is how I contracted the indices (one at a time):
`LEG1 = Contract[TAUk1k2m1*gravprop1] // Simplify
LEG2 = Contract[TAUk3k4m2*gravprop2] // Simplify`

I wrote the 1PI as the following:
`ONEPI = (-G*
   n)/\[Pi]*(1/
     20*(FVD[q, \[Alpha]] FVD[q, \[Beta]] - 
       SPD[q] MTD[\[Alpha], \[Beta]])*(FVD[q, \[Gamma]] FVD[
         q, \[Delta]] - SPD[q] MTD[\[Gamma], \[Delta]]) + 
    1/30*(FVD[q, \[Alpha]] FVD[q, \[Gamma]] - 
       SPD[q] MTD[\[Alpha], \[Gamma]])*(FVD[q, \[Beta]] FVD[
         q, \[Delta]] - SPD[q]*MTD[\[Beta], \[Delta]]) + 
    1/30*(FVD[q, \[Alpha]] FVD[q, \[Delta]] - 
       SPD[q] MTD[\[Alpha], \[Delta]])*(FVD[q, \[Beta]] FVD[
         q, \[Gamma]] - SPD[q] MTD[\[Beta], \[Gamma]]))*Log[-SPD[q]]`

And contracted in a similar manner:
`AMP = Contract[LEG1*LEG2*ONEPI] // Simplify`

The nonrelativistic limit, or the IR limit I wrote as the following:
`SPD[Subscript[k, 1], Subscript[k, 2]] = 
 SPD[Subscript[k, 2], Subscript[k, 2]] = 
  SPD[Subscript[k, 1], Subscript[k, 1]] = (Subscript[m, 1])^2
SPD[Subscript[k, 3], Subscript[k, 3]] = 
 SPD[Subscript[k, 3], Subscript[k, 4]] = 
  SPD[Subscript[k, 4], Subscript[k, 4]] = (Subscript[m, 2])^2
SPD[Subscript[k, 1], Subscript[k, 3]] = 
 SPD[Subscript[k, 1], Subscript[k, 4]] = 
  SPD[Subscript[k, 2], Subscript[k, 3]] = 
   SPD[Subscript[k, 4], Subscript[k, 
     2]] = (Subscript[m, 1])*(Subscript[m, 2])
SPD[q, Subscript[k, 2]] = 
 SPD[Subscript[k, 1], q] = 
  SPD[Subscript[k, 3], q] = SPD[q, Subscript[k, 4]] = 0
SPD[q, q] = -(Q)^2
SPD[SPD[q], SPD[q]] = +(Q^2)^2`

Which if you then ask for AMP again and then plug in D = 4 and kappa^2 = 32\pi G, you get a factor of 2.... not 8/5...

ANY help is greatly appreciated.

**Comment by vsht** [comment]
I don't see anything wrong with the code itself (except for `SPD[SPD[q], SPD[q]] = +(Q^2)^2`, this cannot really occur) so most likely it must be an issue with the entered amplitudes.
